stix_shifter/stix_translation/src/modules/bigfix/json/to_stix_map.json

```diff
@@ -73,9 +102,70 @@
             "cybox": false
         }
     ],
-    "computer_identity":
-    {
+    "computer_identity": {
```

Comment:
I know this has been there before. we need to set `computer_identity` field inside a custom stix object instead of setting as name object. bigfix custom stix object can be named as `x_com_bigfix_relevance`. there are some examples of how to add a custom object in QRadar and Splunk  to_stix_map.json